lib/src/localization.dart

```diff
@@ -218,6 +222,24 @@ class TimetableLocalizationEn extends TimetableLocalizations {
       'Week ${week.weekOfYear}, ${week.weekBasedYear}';
 }
 
+class TimetableLocalizationCs extends TimetableLocalizations {
+  const TimetableLocalizationCs();
+
+  @override
+  List<String> weekLabels(Week week) {
+    return [
+      weekOfYear(week),
+      'Týden ${week.weekOfYear}',
+      'T ${week.weekOfYear}',
+      '${week.weekOfYear}',
+    ];
+  }
+
+  @override
+  String weekOfYear(Week week) =>
+      'Týden ${week.weekOfYear}, ${week.weekBasedYear}';
+}
```

Comment:
Please move this class above `TimetableLocalizationDe` to keep them ordered alphabetically